assets/src/application/services/utilities/date/diff.ts

```diff
@@ -74,8 +74,10 @@ const diff = (
 	dirtyOptions?: DirtyOptions
 ): number => {
 	const func = diffMapping[unit];
+	const diffOutput = unit === 'calendarWeeks' ? func(dateLeft, dateRight, dirtyOptions) : func(dateLeft, dateRight);
 
-	return unit === 'calendarWeeks' ? func(dateLeft, dateRight, dirtyOptions) : func(dateLeft, dateRight);
+	// This approach prevents having two zeros: 0 and -0; more info here: https://2ality.com/2012/03/signedzero.html
+	return Number(diffOutput.toString());
```

Comment:
Fix for:

![image](https://user-images.githubusercontent.com/1477453/79857783-f97a4000-83d6-11ea-8d61-977523306895.png)